src/worker/main.ts

```diff
@@ -23,15 +23,15 @@ const client = new Redis(getRedisInfo("client").redisOptions);
 const subscriber = new Redis(getRedisInfo("subscriber").redisOptions);
 const logger = getLogger("worker.main");
 
-function measureElapsedTime(startTime, tags) {
-	const endTime = Date.now();
-	const timeDiff = endTime - startTime;
-	statsd.histogram(metricHttpRequest().jobDuration, timeDiff, tags);
+function measureElapsedTime(job: Queue.Job, tags) {
+	statsd.histogram(metricHttpRequest().jobDuration, job.finishedOn - job.processedOn, tags);
 }
 
 const queueOpts: QueueOptions = {
 	defaultJobOptions: {
-		removeOnComplete: true
+		attempts: 1,
```

Comment:
shouldn't this be a value > 1? To catch some intermittent 5xxs from Jira or similar?

Reply:
yep, your right, I'll fix that.